bigquery_etl/cli/query.py

```diff
@@ -445,6 +445,7 @@ def info(ctx, name, sql_dir, project_id, cost, last_updated):
 def _backfill_query(
     query_file_path,
     project_id,
+    date_partition_parameter,
```

Comment:
maybe we should have set a default for this `date_partition_parameter="submission_date` for backwards compatibility? @ANich